### DISCUSSION TITLE
Why did v2.0.0 drop support for v2.local?

### DISCUSSION THREAD

**Discussion by mceachen** [panva/paseto#17] [Q&A] [answered]
I wasn't using it: just idly curious.

Thanks for Paseto! :100:

**Comment by panva** [comment, marked as answer]
Because it required libsodium, which i'm no longer willing to accept as a dependency. It's too heavy.

**Reply by UnKnoWn-Consortium** [reply]
Just wanna be sure. Is dropping libsodium the sole reason for the change? Thanks.

**Reply by panva** [reply]
Yes.
